purchase_order_approved/views/purchase_order_view.xml

```diff
@@ -0,0 +1,31 @@
+<!-- Copyright 2017 ForgeFlow S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form - purchase_order_approved</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <button name="button_draft" position="attributes">
+                <attribute name="states">cancel,to approve,approved</attribute>
+            </button>
+            <button name="button_approve" position="after">
+                <button
+                    name="button_release"
+                    type="object"
+                    states="approved"
+                    string="Convert to Purchase Order"
+                    class="oe_highlight"
+                />
+            </button>
+            <field name="order_line" position="attributes">
+                <attribute
```

Comment:
@SurachetS @kittiu @HviorForgeFlow @JordiBForgeFlow Do you agree we need a purchase_stock_order_approved as the readonly should take into account service product as in:

https://github.com/odoo/odoo/blob/14.0/addons/purchase_stock/views/purchase_views.xml#L48

To be able to modify received quantities for service products

Reply:
Ok, found. The 'purchase' state should not be included in the readonly domain. I will do a PR for that